blocklist-control-plan.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 优化规则与表达式下推的黑名单
 summary: 了解优化规则与表达式下推的黑名单。
-aliases: ['/docs-cn/dev/blacklist-control-plan/']
+aliases: ['/docs-cn/dev/blocklist-control-plan/','/zh/tidb/dev/blocklist-control-plan']
```

Comment:
```suggestion
aliases: ['/docs-cn/dev/blacklist-control-plan/','/zh/tidb/dev/blacklist-control-plan']
```

Reply:
Why need to change `blocklist-control-plan` back to `blacklist-control-plan`?

Reply:
This PR deletes `blacklist-control-plan` and adds `blocklist-control-plan`. To make sure the original `blacklist-control-plan` link does not prompt 404 error, so we need to add aliases in the metadata part of the doc. Aliases are only for **doc links TO BE DELETED NOT the doc links TO BE USED**.  In this way, if you  kept the old link of '/zh/tidb/dev/blacklist-control-plan' or '/docs-cn/dev/blacklist-control-plan/' and access them, you will be redirected to the new address '/zh/tidb/dev/blocklist-control-plan'. Otherwise, if we add `blocklist-control-plan` in the aliases, we will bump 404 error. Is my explanation clear?

Reply:
OK